### PR TITLE
Content updates to Security page

### PR DESCRIPTION
**Pull request by anikbrazeau**
Clarifying GC Notify's security information as clients often ask if it meets their security standards.

This includes a page review and new information about:
- Amazon Web Services cloud hosting in Canada
- Shared Services Canada guardrails for Protected B medium availability, medium integrity
- Data storage for only as long as it needs to, default 7 days
- Ways we manage risks include third-party audits and penetration testing
- Messages should not include Protected B information as they are not end-to-end encrypted

### CODE REVIEW

**Comment by anikbrazeau** [comment]
[Update current peripheral content -- Terms, Privacy, Security](https://trello.com/c/h5h9xPAz/600-update-current-peripheral-content-terms-privacy-security)

File: app/templates/views/security.html [hunk 1]
```diff
@@ -109,6 +113,10 @@ <h2 class="heading-medium">{{ _('Data Categorization') }}</h2>
     <p>
       {{ _('You can use GC Notify to send messages designated up to and including “Protected A” per the <a href=https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32614>Standard on Security Categorization</a>.') }}
     </p>
+    <p>
+      {{ _('Messages should not include “Protected B” information as both emails and text messages sent to the public are not end-to-end encrypted.
+') }}
```
Comment (anikbrazeau) [inline app/templates/views/security.html:118]:
```suggestion
      {{ _('Messages should not include “Protected B” information as both emails and text messages sent to the public are not end-to-end encrypted.') }}
```

File: app/templates/views/security.html [hunk 2]
```diff
@@ -48,16 +49,17 @@ <h2 class="heading-medium">{{ _('Technical security') }}</h2>
         </ul>
       </li>
       <li>{{ _('protective monitoring to record activity, and raise alerts about any suspicious activity') }}</li>
-      <li>{{ _('using JSON Web Tokens, to avoid sending API keys when your service talks to GC Notify') }}</li>
+      <li>{{ _('the option to use JSON Web Tokens instead of API keys when your service talks to GC Notify') }}</li>
```
Comment (sastels) [inline app/templates/views/security.html:52]: should these lines have periods at the end?

**Review by sastels — APPROVED** [review]
minor comment, LGTM

File: app/templates/views/security.html (hunk 2 above)
Comment (anikbrazeau) [reply app/templates/views/security.html]: Good catch! Thanks

**Comment by anikbrazeau** [comment]
[Update current peripheral content for accuracy and scannability -- Terms, Privacy, Security](https://trello.com/c/h5iWftWx/414-update-current-peripheral-content-for-accuracy-and-scannability-terms-privacy-security)

**Pull request merged**